src/navargus/glue.py

```diff
@@ -567,9 +568,15 @@ def get_api_timeout(self) -> float:
         """Returns the configured API request timeout value"""
         return float(self.get("api", {}).get("timeout", 2.0))
 
-    def get_sync_on_startup(self):
-        """Returns True if this program should always sync the Argus API on startup"""
-        return bool(self.get("api", {}).get("sync-on-startup"))
+    def get_sync_interval(self):
+        """
+        Returns the interval in minutes of how often the Argus API is synced if
+        it is None, no re-sync should happen
+        """
+        sync_interval = self.get("api", {}).get("sync-interval", 1)
+        if sync_interval is None:
+            return None
+        return int(sync_interval)
```

Comment:
What happens if a user configures the value to 0?  A negative value?  A non-integer value?  Sounds like a bad time.

I think `0` should be considered the same as `None` here (i.e. instead have `if not sync_interval:`).  Also, I assume it is meant to otherwise be a positive integer, so if it cannot be coerced to that, this should raise an exception.